src/domain/locking/entities/campaign.entity.ts

```diff
@@ -0,0 +1,4 @@
+import { CampaignSchema } from '@/domain/locking/entities/schemas/campaign.schema';
+import { z } from 'zod';
+
+export type Campaign = z.infer<typeof CampaignSchema>;
```

Comment:
In some places we locate the schema with the entity as it's a one liners. Should we do the same here?

Unrelated to this PR, we should also do the same everywhere else.

Reply:
Agreed, changed in [9b82116](https://github.com/safe-global/safe-client-gateway/pull/1543/commits/9b821161c6a8779bba527895f7d11893fc500ad5)